components/raftstore/src/store/worker/region.rs

```diff
@@ -692,8 +747,13 @@ where
             if self.ctx.ingest_maybe_stall() {
                 break;
             }
-            if let Some(Task::Apply { region_id, status }) = self.pending_applies.pop_front() {
-                self.ctx.handle_apply(region_id, status);
+            if let Some(Task::Apply {
+                region_id,
+                status,
+                peer_id,
+            }) = self.pending_applies.pop_front()
+            {
+                self.ctx.handle_apply(region_id, status, peer_id);
```

Comment:
The order should be `region_id, peer_id, status`.

Reply:
OK, and it will also affects `apply_snap`